### PR TITLE
Update setupscripts

### PR DESCRIPTION
**Pull request by karel1980**
Use avdmanager and sdkmanager instead of the old 'android' script.
For more context, see  https://github.com/NativeScript/functional-tests-core/pull/48

### CODE REVIEW

File: scripts/android-sdk-update.sh [hunk 1]
```diff
@@ -2,21 +2,33 @@
...
+
+PACKAGES=()
+PACKAGES+=("platform-tools")
+PACKAGES+=("tools")
+PACKAGES+=("build-tools;27.0.3")
+PACKAGES+=("extras;android;m2repository")
+PACKAGES+=("extras;google;m2repository")
+PACKAGES+=("platforms;android-7")
+PACKAGES+=("platforms;android-8")
+PACKAGES+=("platforms;android-9")
```
Comment (vchimev) [inline scripts/android-sdk-update.sh]: `android-7`, `android-8` and `android-9` are not necessary

File: scripts/android-emulator-re-create.sh [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
-echo no | $ANDROID_HOME/tools/android create avd -n Emulator-Api24-Default -t android-24 --abi default/x86 -c 12M -f
+recreate_emulator Emulator-Api17-Default "system-images;android-17;default;x86"
+recreate_emulator Emulator-Api18-Default "system-images;android-18;default;x86"
+recreate_emulator Emulator-Api19-Default "system-images;android-19;default;x86"
+recreate_emulator Emulator-Api21-Default "system-images;android-21;default;x86"
+recreate_emulator Emulator-Api22-Default "system-images;android-22;default;x86"
+recreate_emulator Emulator-Api23-Default "system-images;android-23;default;x86"
+recreate_emulator Emulator-Api24-Default "system-images;android-24;default;x86"
 
 echo "ARM Emulators"
```
Comment (vchimev) [inline scripts/android-emulator-re-create.sh]: I would keep only `default;x86` and `google_apis;x86 ` emulators and would remove `armeabi-v7a` from the script.

File: scripts/android-emulator-images-update.sh [hunk 3]
```diff
@@ -2,46 +2,67 @@
...
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-google_apis-21 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-armeabi-v7a-android-21 --all --no-ui
+echo "Update Intel HAXM"
+echo y | "${ANDROID_HOME}/tools/bin/sdkmanager" "extras;intel;Hardware_Accelerated_Execution_Manager"
+sudo sh "${ANDROID_HOME}/extras/intel/Hardware_Accelerated_Execution_Manager/silent_install.sh"
 
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-android-19 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-armeabi-v7a-android-19 --all --no-ui
-echo y | $ANDROID_HOME/tools/android update sdk --filter sys-img-x86-google_apis-19 --all --no-ui
+PACKAGES=()
```
Comment (vchimev) [inline scripts/android-emulator-images-update.sh]: I would keep only the emulator images that are used in the create script and remove the rest to save disk space.

**Comment by vchimev** [comment]
Hey @karel1980,

Thank you very much for this pull request.

I would be happy to merge it as is and provide a full list of emulator images. However, please, let me know what do you think on my comments.

File: scripts/android-sdk-update.sh (hunk 1 above)
Comment (karel1980) [reply scripts/android-sdk-update.sh]: Will remove those

File: scripts/android-emulator-re-create.sh (hunk 2 above)
Comment (karel1980) [reply scripts/android-emulator-re-create.sh]: ok

File: scripts/android-emulator-images-update.sh (hunk 3 above)
Comment (karel1980) [reply scripts/android-emulator-images-update.sh]: ok

File: scripts/android-emulator-re-create.sh (hunk 2 above)
Comment (karel1980) [reply scripts/android-emulator-re-create.sh]: Will remove arm64-v8a as well

Comment (karel1980) [reply scripts/android-emulator-re-create.sh]: ... and the mips one

**Comment by karel1980** [comment]
@vchimev rework done. Please have another look

**Comment by karel1980** [comment]
Just one more remark/question: I've also removed the x86_64 system images. I hope that's allright.

**Comment by vchimev** [comment]
Congratulations on this quality PR! Have a badge!
contrib

**Comment by karel1980** [comment]
Haha, thanks.

**Pull request merged**